.github/workflows/ci.yaml

```diff
@@ -106,14 +112,19 @@ jobs:
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
 
       - name: Run cargo build with devnet-prealloc feature
-        run: cargo build --features devnet-prealloc --workspace --all --tests --benches
-        
+        run: cargo build --release --features devnet-prealloc --workspace --all --tests --benches
+
       - name: Run cargo test regular features
-        run: cargo test --workspace
+        run: cargo nextest run --release --workspace
+
+      - name: Run cargo doc tests with features=no-asm
+        run: cargo test --doc --release --workspace
 
       - name: Run cargo test on kaspa-hashes without asm
-        run: cargo test -p kaspa-hashes --features=no-asm --benches
+        run: cargo nextest run --release -p kaspa-hashes --features=no-asm --benches
 
+      - name: Run cargo doc tests with features=no-asm
```

Comment:
add "on kaspa-hashes"

Reply:
We can't add it since `-p kaspa-hashes` is not an option we can add when doing `--doc`

Reply:
Nvm, `cargo test --doc --release -p kaspa-hashes --features=no-asm` would actually work. but with this result:

```
test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s
```

Assuming that in the future this may have docs, I'll add this step in.